pkg/kv/dump.go

```diff
@@ -141,6 +141,17 @@ var SectionMapping = map[string][]string{
 	"kv":    {"kv-internal-metadata"},         // kv internal metadata
 }
 
+// GetAllKnownPartitions returns all partitions defined in SectionMapping
```

Comment:
Why do we need this?  CreateDump _still_ tests `len(partitions) == 0` and collects all known partitions.